streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -762,7 +1665,7 @@
      * <td>&lt;K1:A&gt;</td>
      * <td></td>
      * <td></td>
-     * <td></td>
+     * <td>&lt;K1:ValueJoiner(A,null)&gt;</td>
```

Comment:
same here

Reply:
I think, these changes come from javadoc shift due to new methods added.